### PR TITLE
[CHF-112] Add Health Check capability to SmartThings SmartSense products

### PR DESCRIPTION
**Pull request by jackchi**
By adding `capability Health Check` Device Watch will monitor Devices send `online:offline` events to the `Location`

https://smartthings.atlassian.net/browse/CHF-112

### CODE REVIEW

**Comment by jackchi** [comment]
cc @darksun

File: devicetypes/smartthings/arrival-sensor.src/arrival-sensor.groovy [hunk 1]
```diff
@@ -19,6 +19,7 @@ metadata {
 		capability "Presence Sensor"
 		capability "Sensor"
 		capability "Battery"
+		capability "Health Check"
```
Comment (darksun) [inline devicetypes/smartthings/arrival-sensor.src/arrival-sensor.groovy]: Let's remove for this dth.

**Comment by darksun** [comment]
👍

**Comment by jackchi** [comment]
All the new Samjin device types that SmartThings produces is now: `SmartSense-xxxx-sensor.groovy`. 

6 DTHs updated:

Smartsense Motion Sensor
Smartsense Multi Sensor 
Smartsense Open Close Sensor 
Smartsense Temp-Humidity 
Smartsense Water Leak Sensor 
Smartthings Outlet

**Pull request merged**